.github/workflows/packer.yml

```diff
@@ -41,45 +41,56 @@ jobs:
         patch-wording:  'patch,fixes'     # Providing patch-wording will override commits
                                           # defaulting to a patch bump.
         rc-wording:     'RELEASE,alpha'
-        skip-commit: 'true'
         skip-tag: 'true'
-        skip-push: 'true'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
     - name: Set environment variables for images
       run: |
         # Get the branch name, remove 'refs/heads/' from github.ref
         BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}" | tr '/' '-')
-        # If the branch is 'main', don't append the branch name to the tag
-        if [ "$BRANCH_NAME" == "main" ]; then
-          TAG=${{ steps.bump_version.outputs.newTag }}
-        elif [[ "${{ github.event_name }}" == "release" ]]; then
+        GIT_SHA=$(echo "$(git rev-parse --short HEAD)")
+        GIT_BRANCH=$(echo "$(git rev-parse --abbrev-ref HEAD)")
+        # if its a release event bump the version and use that as tag
+        if [[ "${{ github.event_name }}" == "release" ]]; then
+          # In case of release event the tag will be the next version
+          # build version will be the same as tag
           TAG=${{ steps.bump_version.outputs.newTag }}
+          BUILD_VERSION=${TAG}
+          RELEASE_EVENT=true
```

Comment:
I am not able find usage of this `RELEASE_EVENT` can you add a comment for the usage?